StaticAnalyzer/views/android/java.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf_8 -*-
+"""Module for java analysis."""
+
+import re
+import shutil
+import os
+
+from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from django.conf import settings
+from django.utils.html import escape
+
+from MobSF.utils import (
+    PrintException
+)
+
+def run(request):
+    """Perform static analysis on java code."""
```

Comment:
We don't perform static code analysis here. It's just listing java source code files.